### ISSUE TITLE
- nothing provides kubernetes-cni needed by cri-o-1.28.1-150500.1.1.x86_64

### ISSUE THREAD

**Issue by alanpeng** [cri-o/packaging#36]
When added the CRI-O repo as below:
[cri-o]
name=CRI-O
baseurl=https://pkgs.k8s.io/addons:/cri-o:/stable:/v1.28/rpm/
enabled=1
gpgcheck=1
gpgkey=https://pkgs.k8s.io/addons:/cri-o:/stable:/v1.28/rpm/repodata/repomd.xml.key

I got the error message when execute the command yum install cri-o:

Importing GPG key 0x9A296436:
 Userid     : "isv:kubernetes OBS Project <isv:kubernetes@build.opensuse.org>"
 Fingerprint: DE15 B144 86CD 377B 9E87 6E1A 2346 54DA 9A29 6436
 From       : https://pkgs.k8s.io/core:/stable:/v1.28/rpm/repodata/repomd.xml.key
Key imported successfully
Complete!
The downloaded packages were saved in cache until the next successful transaction.
You can remove cached packages by executing 'yum clean packages'.
AlmaLinux 9 - AppStream                          59 MB/s |  11 MB     00:00    
AlmaLinux 9 - BaseOS                             41 MB/s | 4.7 MB     00:00    
AlmaLinux 9 - Extras                            201 kB/s |  18 kB     00:00    
CRI-O                                           9.9 kB/s | 2.0 kB     00:00    
Error: 
 Problem: cannot install the best candidate for the job
  - nothing provides kubernetes-cni needed by cri-o-1.28.1-150500.1.1.x86_64
(try to add '--skip-broken' to skip uninstallable packages or '--nobest' to use not only best candidate packages)

Exited with code exit status 1

**Comment by haircommander** [comment]
hm interesting the `Recommends` clause should mean it's not required

**Comment by saschagrunert** [comment]
v1.28.1 is unfortunately broken, there is no stable release available yet via this repository. v1.28.2 will fix this.

**Comment by saschagrunert** [comment]
Let’s keep it open to verify.

**Comment by afbjorklund** [comment]
> hm interesting the `Recommends` clause should mean it's not required

The "kubernetes-cni" package is provided by a different repository...

It was working in Fedora, due to a weird "Provides" on a system package:

```
containernetworking-plugins-1.1.1-16.fc38.x86_64 : Libraries for writing CNI plugin
Repo        : fedora
Matched from:
Provide    : kubernetes-cni
```

Which was unrelated anyway, since it provides `/usr/lib/cni` but **not** `/opt/cni/bin`.

**Comment by afbjorklund** [comment]
Fixed in cri-o-1.29.0-150500.1.1.x86_64

**Comment by saschagrunert** [comment]
Closing this one for https://github.com/cri-o/packaging/issues/68 and https://github.com/cri-o/packaging/issues/67
